pages/dkp/kommander/2.2/install/configuration/enterprise-catalog/index.md

```diff
@@ -46,13 +46,15 @@ The following section describes each label:
 
 ### Air-gapped Catalog Configuration
 
-When running in air-gapped environments, update the configuration by replacing `gitRepositorySpec` with the `path` field pointing to a local path of the catalog git repository folder, for example:
+When running in air-gapped environments, update the configuration by replacing `gitRepositorySpec` with the `path` field pointing to a local path of the DKP catalog applications git repository.
+
+1. Download the DKP catalog application Git repository archive:
 
 ```bash
-git clone https://github.com/mesosphere/dkp-catalog-applications
+wget "https://downloads.d2iq.com/dkp/${VERSION}/dkp-catalog-applications-${VERSION}.tar.gz"
```

Comment:
Does the `path` in the config snippet below need to be updated to point to this tarball?

Reply:
good call! I'll have to test if it will accept a tarball or if we have to untar it. maybe @takirala knows off the top of his head.

Reply:
tested this locally quick and using the tar worked fine 🎉 

```
uploading chartbundle from "dkp-catalog-applications-charts.tar.gz" to chartmuseum
chart kafka-operator-0.20.0.tgz uploaded to chartmuseum from charts-bundle
chart kafka-operator-0.20.2.tgz uploaded to chartmuseum from charts-bundle
chart spark-operator-1.1.17.tgz uploaded to chartmuseum from charts-bundle
chart spark-operator-1.1.6.tgz uploaded to chartmuseum from charts-bundle
chart zookeeper-operator-0.2.13.tgz uploaded to chartmuseum from charts-bundle
```
pushed up c00442a8c4657a7adfe63a24d3a430281c0e72ab to fix it

Reply:
thanks @branden! after https://github.com/mesosphere/dkp-catalog-applications/pull/62, using the tarball works just fine